homeassistant/components/launch_library/sensor.py

```diff
@@ -84,57 +141,56 @@ async def async_setup_entry(
     async_add_entities(
         [
             NextLaunchSensor(
-                coordinator,
-                entry.entry_id,
-                name,
-                SENSOR_DESCRIPTION,
-            ),
+                coordinator=coordinator,
+                entry_id=entry.entry_id,
+                description=description,
+                name=name if description.name == NEXT_LAUNCH else None,
```

Comment:
Is this the correct way to make sure we use the name people already have setup in the old version?

Reply:
This works :+1:  it's only until we remove config import anyways.